README.md

```diff
@@ -177,10 +177,10 @@ Press `Ctrl-Shift-P` to open the command palette and type `Material Icons`.
 | **Change Folder Theme**           | Change the design of the folder icons.                                             |
 | **Change Opacity**                | Change the opacity of the icons.                                                   |
 | **Change Saturation**             | Change the saturation value of the icons.                                          |
-| **Configure Icon Packs**          | Select an icon pack that enables additional icons (e.g. for Angular, React, Ngrx). |
-| **Hide Folder Arrows**            | Hides the arrows next to the folder icons.                                         |
-| **Restore Default Configuration** | Reset the default configurations of the icon theme.                                |
-| **Toggle Grayscale**              | Sets the saturation of the icons to zero, so that they are grayscale.              |
+| **Configure Icon Packs**          | Selects an icon pack that enables additional icons (e.g. for Angular, React, Ngrx).|
+| **Toggle Explorer Arrows**        | Show or hide the arrows next to the folder icons.                                  |
+| **Restore Default Configuration** | Reset to the default configuration.                                                |
+| **Toggle Grayscale**              | Set icon saturation to `0` (grayscale), or `1` (colour).                           |
```

Comment:
I'd prefer it if you could use `color` instead of `colour`. It's due to consistency to the other parts of the description, where I prefer American English ;)

Reply:
Sorry, yes, I totally forgot to ask if this was OK! I’ll change it.